internal/datastore/post.go

```diff
@@ -70,6 +71,90 @@ func (d *delphisDB) GetPostsByDiscussionIDIter(ctx context.Context, discussionID
 	}
 }
 
+/* Equivalent of GetPostsByDiscussionIDIter, but accepting a cursor and a limit for fetching. In our implementation,
```

Comment:
I created a new method instead of modifying GetPostsByDiscussionIDIter signature in order to not break old code dependencies. Also, their use cases can effectively be different, considering that they rely on different SQL statements.